File: Curso Java Com POO/Java com POO/Aula12e13/src/aula12e13/Ave.java
package aula12e13;

public class Ave extends Animal {
    //Atributos
    private String corPena;
    
    //Métodos
    public void fazerNinho() {
        System.out.println("Ave está fazendo seu ninho!");
    }
    
    //Métodos Sobrepostos
    @Override
    public void locomover() {
        System.out.println("Voando");
    }

    @Override
    public void alimentar() {
        System.out.println("Comendo frutas");
    }

    @Override
    public void emitirSom() {
        System.out.println("Som de Ave");
    }
    
    //Getters e Setters
    //corPena
    public String getCorPena() {
        return corPena;
    }
    public void setCorPena(String corPena) {
        this.corPena = corPena;
    }
    
    
}

File: Curso Programação Orientada a Objetos/PHP com POO/Aula12/Ave.php
<?php

require_once 'Animal.php';

class Ave extends Animal{
    //Atributos
    private $corPena;
    
    //Métodos
    public function fazerNinho() {
        echo "<p>Ave está fazendo seu ninho!</p>";
    }
    
    //Métodos Sobrepostos
    public function locomover() {
        echo "<p>Voando</p>";
    }

    public function alimentar() {
        echo "<p>Comendo frutas</p>";
    }

    public function emitirSom() {
        echo "<p>Som de ave</p>";
    }
    
    //Getters
    public function getCorPena() {
        return $this->corPena;
    }

    //Setters
    public function setCorPena($corPena) {
        $this->corPena = $corPena;
    }


}
